src/app/projects/new-project/new-project.component.ts

```diff
@@ -0,0 +1,73 @@
+import { ChangeDetectorRef, Component, OnDestroy, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup } from "@angular/forms";
```

Comment:
There should be quotes instead of double quotes.

Reply:
done